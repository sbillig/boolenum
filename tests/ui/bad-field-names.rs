use boolenum::BoolEnum;

#[derive(BoolEnum)]
pub enum FieldNames {
    Foo,
    Bar,
}

fn main() {}
