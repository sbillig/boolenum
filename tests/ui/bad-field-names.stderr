error: BoolEnum can only be used on enums with two variants named No and Yes, or False and True.

  = help: try `enum FieldNames { No, Yes }`

 --> $DIR/bad-field-names.rs:4:10
  |
4 | pub enum FieldNames {
  |          ^^^^^^^^^^
